libr/core/graph.c

```diff
@@ -3571,6 +3571,8 @@ static void rotateColor(RCore *core) {
 R_API int r_core_visual_graph(RCore *core, RAGraph *g, RAnalFunction *_fcn, int is_interactive) {
 	int o_asmqjmps_letter = core->is_asmqjmps_letter;
 	int o_scrinteractive = r_config_get_i (core->config, "scr.interactive");
+	bool asm_pseudo = r_config_get_i (core->config, "asm.pseudo");
```

Comment:
see if you can use RConfigHold instead of the usual save/set/restore which is more error prone. It should be enough to call either `r_config_save_num` or `r_config_save_char` and r_config_restore at the end of this function should do everything else.